app/javascript/packs/pages/volunteer_signup/VolunteerStepper.jsx

```
@@ -196,7 +198,7 @@ export default function VolunteerStepper({ steps, location }) {
           }
         })
         .catch((e) => {
-          setErrors([e.message]);
+          setErrors([errorMessage.missingLocation]);
```

Does this override other error messages, like missing email? We might need to map other possible error messages too.

---

Or I haven't looked into it, but maybe just the location error message can be changed on the back end?

---

Yes, this looks like it might hide other error messages?  Amy, I just made a PR that should fix this issue on the backend instead: https://github.com/feedadoc/feedadoc/pull/105

Thank you for working on this!

---

Awesome! I'll take a look at that PR